lib/parser.js

```diff
@@ -627,29 +627,32 @@ function handleWow(keys)
   expectAnyToken(['wow', 'wow&'], keys);
   multiLine = false;
 
+  var chained = keys[0] === 'wow&';
+
+  // consume: wow/wow&
+  keys.shift();
+
   var statement = '';
 
   // add keys if there's a return value
-  if (typeof keys[1] !== 'undefined') {
+  if(keys.length > 0)
+  {
     statement += 'return';
-
-    for(var i = 1; i < keys.length; i++)
+    // ideally this will call parseKeys in the future to support return foo();
```

Comment:
i'm hoping soonish we'll be able to do:
```
wow plz foo
``` 

with this call just calling `parse` or `parseTokens(keys)`